frontend/src/metabase/admin/datamodel/components/FilterPopover/FilterPopover.tsx

```diff
@@ -286,7 +286,6 @@ export function FilterPopover({
           className={className}
           filter={filter}
           dateShortcutOptions={dateShortcutOptions}
-          primaryColor={primaryColor}
```

Comment:
Remove runtime color since it'll just be resolved to the same color as `var(--mb-color-text-brand)`